### PR TITLE
Prevent initial animation

### LINKED ISSUE #1: Don't animate the initial color
If you launch the Example project, you can see that it animates the border from black to pink quickly at startup. It should not animate initially.

### PR DESCRIPTION
**Pull request by phjs**
I have ensured all button properties are set during setup and only animated properties are set/changed during animation. This prevents the animation of the initial color as described in your [issue](https://github.com/sindresorhus/CustomButton/issues/1).

Fixes #1

### CODE REVIEW

File: Sources/CustomButton/CustomButton.swift [hunk 1]
```diff
@@ -24,97 +24,87 @@ open class CustomButton: NSButton {
...
-	@IBInspectable public var cornerRadius: Double = 0 {
+	@IBInspectable public var cornerRadius: CGFloat = 0 {
 		didSet {
-			needsDisplay = true
+            layer?.cornerRadius = cornerRadius
 		}
 	}
 
-	@IBInspectable public var borderWidth: Double = 0 {
+	@IBInspectable public var borderWidth: CGFloat = 0 {
```
Comment (sindresorhus) [inline Sources/CustomButton/CustomButton.swift]: Why are you changing this and the other types? I would prefer to leave this as-is unless there's a good reason to change them.

**Comment by sindresorhus** [comment]
Thanks for working on this 🙌

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: Sources/CustomButton/CustomButton.swift (hunk 1 above)
Comment (phjs) [reply Sources/CustomButton/CustomButton.swift]: The reasons were: 
1) the properties don't need the higher precision of `Double` and
2) it reduces the amount casts (to zero), which in return increases the readability.

If you insist on the initial data types, I'll gladly change the code back

Comment (sindresorhus) [reply Sources/CustomButton/CustomButton.swift]: It is generally nicer to use `Double` everywhere. CGFloat is Double anyway on macOS, so the extra type is just annoying as a user.

> it reduces the amount casts (to zero), which in return increases the readability.

I'd rather reduce the readability here for a nicer API. Reduce the readability once and every user doesn't have to convert their Double's to CGFloat/Float.

> If you insist on the initial data types, I'll gladly change the code back

I insist. It's just how I prefer my APIs to be. Thanks for understanding.

Comment (phjs) [reply Sources/CustomButton/CustomButton.swift]: > It is generally nicer to use Double everywhere. CGFloat is Double anyway on macOS, so the extra type is just annoying as a user.

Good to know, I only knew it where a typedef for `Float`

**Comment by sindresorhus** [comment]
Looks great 👌

**Comment by phjs** [comment]
Thanks for fixing the indents, I am wondering why they were so broken

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/CustomButton/releases/tag/v1.0.0

**Pull request merged**